### PR TITLE
fix(designer): Revert of Add UI for invoker connection support (#1990)

### PR DESCRIPTION
**Pull request by ccastrotrejo**
### Main code changes
- This PR [Add UI for invoker connection support (#1990)](https://github.com/Azure/LogicAppsUX/pull/1990) introduced changes that caused issues when user wants to select dynamic data from dropdown

The issue was detected while using the `Transform using Data Mapper XSLT` connector. When trying to select map name from the dropdown, no option gets rendered. 

### Without isssue 

https://user-images.githubusercontent.com/102700317/232141073-0cc378ed-d305-4718-a59e-5dca8c52661e.mov

### Issue

https://user-images.githubusercontent.com/102700317/232141137-8157ccba-4d92-45fb-a8b7-db379c09e37c.mov

### CODE REVIEW

**Comment by ccastrotrejo** [comment]
Hi @bnarayanared we are reverting the changes from this [PR](https://github.com/Azure/LogicAppsUX/pull/1990)

**Comment by bnarayanared** [comment]
You don't have to revert the entire PR, you can just undo the changes in files:
1. libs/designer/src/lib/core/queries/connector.ts
2. libs/designer/src/lib/core/utils/parameters/dynamicdata.ts

I don't know how those got checked in, These changes are in a separate PR that isn't checked-in

File: libs/designer/src/lib/core/queries/connector.ts
Comment (bnarayanared) [inline libs/designer/src/lib/core/queries/connector.ts:1]: Undo changes on this file

File: libs/designer/src/lib/core/utils/parameters/dynamicdata.ts
Comment (bnarayanared) [inline libs/designer/src/lib/core/utils/parameters/dynamicdata.ts:1]: Undo changes on this file

Comment (hartra344) [reply libs/designer/src/lib/core/utils/parameters/dynamicdata.ts:1]: This is remaining a full reversion until we can get some more testing. We did a new product release today and this broke it. We need to prioritize getting that hot fixed.

**Review by hartra344 — APPROVED** [review]

**Pull request merged**